vagrant/bootstrap.sh

```diff
@@ -34,7 +36,7 @@ sudo chown $FFNM_USERNAME .
 ffsudo="sudo -u $FFNM_USERNAME"
 
 : "#### We need some development libraries for the build process:"
-sudo apt update && sudo apt -y --no-install-recommends install git msmtp libc6-dev libssl-dev libpq-dev curl gcc pkg-config
+sudo apt update --allow-releaseinfo-change && sudo apt -y upgrade && sudo apt -y --no-install-recommends install git msmtp libc6-dev libssl-dev libpq-dev curl gcc pkg-config
```

Comment:
"--allow-releaseinfo-change" seems rather suspicious.

Reply:
I had buster 9.99 and there is 10.0 out already and without the option, `apt` cannot update. the option is harmless

Reply:
The option AFAIK was added for security reasons. If you had 9.99 before that sounds like you were using a pre-release of buster or so. Outside dist upgrades, this option should not be needed, so I'd rather not add it here.